tools/nxfuse/README.txt

```diff
@@ -0,0 +1,118 @@
+nxfuse
+======
+
+A Linux FUSE implmentation that enables native mounting of TizenRT filesystems
+in Linux.  Currently the implementation supports SmartFS and NXFFS.  The
```

Comment:
We don't support the NXFFS. Could you remove it ?

Reply:
Done.